app/javascript/dashboard/i18n/locale/en/generalSettings.json

```diff
@@ -92,7 +92,8 @@
   },
   "NETWORK": {
     "NOTIFICATION": {
-      "TEXT": "Disconnected from Chatwoot"
+      "TEXT": "Disconnected from Chatwoot",
```

Comment:
@nithindavid Do we need this i18n here? Are we using it anywhere?

Reply:
@iamsivin I have removed this. Can you take a look again?